restful.entity.inc

```diff
@@ -96,9 +86,11 @@ function restful_entity_update($entity, $type) {
     if (!$handler->hasSimpleInvalidation()) {
       return;
     }
-    $handler
-      ->getCacheController()
-      ->clear($hash);
+    // You can get away without the fragments for a clear.
+    $cache_object = new RenderCache(new ArrayCollection(), $hash, $handler->getCacheController());
+    // Do a clear with the RenderCache object to also remove the cache fragment
+    // entities.
+    $cache_object->clear();
```

Comment:
I was not clearing the render cache properly before. If you do it this way, you clear the caches and delete the involved cache fragments.